include/ceed-impl.h

```diff
@@ -23,12 +23,7 @@
 #include <ceed-backend.h>
 #include <stdbool.h>
 
-#define CEED_INTERN CEED_EXTERN __attribute__((visibility ("hidden")))
-
-#define CEED_MAX_RESOURCE_LEN 1024
-#define CEED_ALIGN 64
-
-#define CEED_COMPOSITE_MAX 16
+#define CEED_EPSILON 1E-16
```

Comment:
@jedbrown, I made this a preprocessor macro for now. I figure we can get fancy when we get around to 
```
/// We can discuss ways to avoid forcing these to be compile-time decisions,
///   but let's leave that for later.
```
for `CeedScalar`